### PR TITLE
Adding GoodVibes to conda-forge

### PR DESCRIPTION
**Pull request by luchini18**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by bobbypaton** [comment]
just confirming I am a willing maintainer...

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/example/meta.yaml:1]: Please do not modify the example `meta.yaml`. If you have suggested changes, they should be submitted in a separate pull request.

File: recipes/GoodVibes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6c7dfef22868f57762e910888e45daf96e2e9cc6cbb7cf299cc6a22cec37ee5c
+
+
+build:
```
Comment (synapticarbors) [inline recipes/GoodVibes/meta.yaml:13]: I believe this could be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

File: recipes/GoodVibes/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy
```
Comment (synapticarbors) [inline recipes/GoodVibes/meta.yaml:19]: I don't think `numpy` isn't required under `host` as it's not a build requirement and is only need to run the package

Comment (luchini18) [reply recipes/GoodVibes/meta.yaml]: Thanks for the help I've added these changes

File: recipes/example/meta.yaml (hunk 1 above)
Comment (synapticarbors) [reply recipes/example/meta.yaml:1]: This was marked as resolved, but I still see modifications to `example/meta.yaml`

Comment (luchini18) [reply recipes/example/meta.yaml]: My mistake, I had added back in an older version, the most recent commit should have the original examples/meta.yaml

**Comment by synapticarbors** [comment]
@conda-forge/staged-recipes -- It looks like this is failing due to a bug in `conda-build` that was patched ~20 days ago:

https://github.com/conda/conda-build/commit/bb8b53f8b3d0c365b54cb7329fd50bc57a5060ff

It looks like it's in the `3.19.3` release. Any idea of when we will be updating?

**Comment by chrisburr** [comment]
See https://github.com/conda-forge/conda-build-feedstock/pull/134#issuecomment-613279587

**Comment by luchini18** [comment]
Just to clarify, are we holding off until this issue is resolved?

**Comment by synapticarbors** [comment]
I think we need to, unfortunately, but maybe another more experienced reviewer might know more.

**Comment by chrisburr** [comment]
If this is blocking something important you could try downgrading conda-build to an older version by editing [this line](https://github.com/conda-forge/staged-recipes/blob/3d66fa63d7f25037fcd9a4074e629663ccf22bbb/.circleci/build_steps.sh#L45) to check the CI works. The change should be reverted before this is merged and the feedstock build will then fail but you can apply a similar hack in the feedstock after it's created.
